### DISCUSSION TITLE
Does Free Trial until April 7, 2025 of https://scala.jfrog.io will be fixed?

### DISCUSSION THREAD

**Discussion by tremblaysimon** [sbt/sbt#7791] [General]
Hello, 

According to https://github.com/sbt/sbt/issues/6294, https://scala.jfrog.io contains the legacy stuff of sbt and sbt plugins. Currently there is a message on top of the site that indicates there is a free trial that will end on April 7, 2025. Do you plan to regulate the situation and maintain https://scala.jfrog.io repositories?
![Screenshot from 2024-10-21 09-35-35](https://github.com/user-attachments/assets/6f50936f-3381-4ed9-8e01-6c18cea3f8a3)

**Comment by adpi2** [comment]
Hello @tremblaysimon,

JFrog has kindly provided us with a free open source plan, which they have been renewing for us on an annual basis. While the message on the site might be a bit misleading, it indicates that we will need to request a renewal before the stated date.

So, to answer your question: yes, we intend to maintain https://scala.jfrog.io/ repositories, or we'll develop a mitigation plan if needed.

**Reply by mkurz** [reply]
I am wondering since a while, why this JFrog thing is even needed. This repo is read only. It's basically just serving files (jars, poms, xml files). Wouldn't it make sense to just crawl (or download from JFrog if possible) all the files, preserving it's folder structure and just serve those files from a virtual machine running a very basic nginx setup (or apache or just write a bash script that acts as file server :wink: just kidding, but I know you can have python scripts with like 20 lines that do exactly that).
And then just leave this vm running for the next 10 years.
So you don't have to deal with JFrog's renewal process. Or am I missing something what JFrog provides us that I just don't have on the radar?

**Reply by adpi2** [reply]
It is not entirely read-only, as we are still publishing debian packages to it. Also we should consider the throughput which is quite high (several TB/year), and the impact on enterprise environments, behind firewalls.
